#2060 Bugfix: Application Process missing from exercise side nav

warrensearle opened the pull request:
## What's included?
Closes #2060 

## Who should test?
✅ Developers

## How to test?
Create a new exercise and check that Application Process appears as a link in the side navigation:
<img width="298" alt="image" src="https://github.com/jac-uk/admin/assets/8524401/222ee9d2-d68d-4ac8-bd92-16bbd6981e39">

@drieJAC please check that this fixes the issue we spotted
@tomlovesgithub @HalcyonJAC please check you are happy with the code changes

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_

HalcyonJAC reviewed (approved):
Looks great to me 👍

nickaddy commented:
I can see in the side bar but not in the main window
![image](https://github.com/jac-uk/admin/assets/138654969/dc3e8651-b148-4857-897a-b29ab550672e)

warrensearle commented:
> I can see in the side bar but not in the main window
Thanks @nickaddy I should have been clearer. There are a few PRs for Application Process: in this one we are fixing a bug that was introduced a month or so ago where the Application Process link in the side navigation was no longer showing up for new exercises. Here we have fixed that. We have another PR in which we've introduced new functionality to include the Application Process in the exercise task list (meaning users must navigate through the Application Process pages in order to get their exercise approved) #2055. Once we've merged in this fix then #2055 will be un-blocked and can be tested (right now there's no way to test as the link to Application Process doesn't show up for new exercises 🤦 )
